### PR TITLE
Create Slides 3 4

### LINKED ISSUE #99: 🚀 Feature: A3: Animation: Slide 3-4
---

This is a :rocket: Feature Request

### PR DESCRIPTION
**Pull request by gettyv**
Closes #99

### CODE REVIEW

File: src/components/Activities/Activity3/Animations/ManyEmployees.tsx [hunk 1]
```diff
@@ -8,32 +8,65 @@ interface ManyEmployeesProps {
...
+    timeline.current?.seek((slideIdx - 1) * 2000);
     if (props.start === true) {
       const timeout = setTimeout(() => {
         timeline.current?.play();
       }, 250);
+
       return () => clearTimeout(timeout);
     }
-  }, [reloadTime, props.start]);
+  }, [reloadTime, props.start, slideIdx]);
```
Comment (BryanPan342) [inline src/components/Activities/Activity3/Animations/ManyEmployees.tsx:67]: is props.start still needed?

if so, we should have a check to see if start is true to start playing and if not then we should pause and clear our timeout!

**Review by BryanPan342 — CHANGES_REQUESTED** [review]
this is a great first stab!

would like to reduce some code duplication and not sure about prop.start

File: src/components/Activities/Activity3/Animations/ManyEmployees.tsx [hunk 2]
```diff
@@ -8,32 +8,65 @@ interface ManyEmployeesProps {
...
+    });
+
+    timeline.current?.add({
+      targets: ['#grouping-teams', '#many-employees'],
+      opacity: [1,0],
+      duration: 1000,
+    }).add({
+      targets: ['#two-employees', '#bubble-1', '#bubble-2'],
+      opacity: [0, 1],
+      duration: 1000,
```
[the remark this answers is not in the document]
Comment (Phreakester) [reply src/components/Activities/Activity3/Animations/ManyEmployees.tsx:36]: Oh, so it kinda appends that to the end of the object? Seems very useful, thank you!

File: src/components/Activities/Activity3/Animations/ManyEmployees.tsx (hunk 1 above)
Comment (Phreakester) [reply src/components/Activities/Activity3/Animations/ManyEmployees.tsx:67]: Hmmm, I believe so as the first slide doesnt have an animation so we dont want to start playing until after.

Comment (Phreakester) [reply src/components/Activities/Activity3/Animations/ManyEmployees.tsx:67]: Unless you see a better way to stop just the first slide from playing?

Comment (BryanPan342) [reply src/components/Activities/Activity3/Animations/ManyEmployees.tsx]: If it's just the first slide I say we just check if slideIdx is 0?

**Review by BryanPan342 — APPROVED** [review]
LGTM

**Pull request merged**